### DISCUSSION TITLE
django filer folder permissions

### DISCUSSION THREAD

**Discussion by wagnerfelix** [django-cms/django-filer#1399] [Ideas]
Hi,

first of all thank you for regularly updating your code.
I would like to request a change for the folder permission admin. We have quite a few permissions and since the list is growing it would be great if the list could be ordered by "pretty_logical_path" and "who". Now i already tried simply adding `ordering = ("pretty_logical_path", )` to the `PermissionAdmin` but that or any other ordering did not have any effect. I guess you can't order by string? Any help would be appreciated.

Regards

**Comment by fsbraun** [comment]
Unfortunately, Django admin does the ordering on queryset level. `pretty_logical_path` is not a database field and also cannot simply be recreated in a queryset since it would require Common Table Expressions (for recursive chaining of the path) which Django does not support out of the box. A similar situation is present in the `who` field.

An alternative might be creating a **custom filter** which would allow to only show certain users, groups, etc. or certain folders. I have not done this, but it might be worth exploring.

@vinitkumar Do you have any thoughts?

**Comment by jrief** [comment]
@wagnerfelix 
django-filer version 4 will offer a new permission system based on ACLs. Here is a proposal on how this might be implemented:
https://github.com/django-cms/django-filer/blob/finder/README-Finder.md#permission-system-proposal

Feel free to comment on this.
